test/test_tableauauth_model.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import warnings
+import tableauserverclient as TSC
+
+
+class TableauAuthModelTests(unittest.TestCase):
+    def setUp(self):
+        self.auth = TSC.TableauAuth('user',
+                                    'password',
+                                    site_id='dad65087-b08b-4603-af4e-2887b8aafc67',
```

Comment:
Wrong site id type still.

This is the thing in the url after `/t/` or `/site/` NOT a luid.

Just make it the string `site1`